### ISSUE TITLE
API Key

### ISSUE THREAD

**Issue by thecaligarmo** [voliva/gw2-pvp-pips#1]
During the last season the API key part was working, but now when I try and add it, it won't actually show anything. Also, when I try and click on "Create new goal" everything disappears and I can't do anything.

**Comment by voliva** [comment]
Hey thanks for the report.

This usually happens when a new season starts, because the API doesn't return the new season yet, and this app expects a season to be active (otherwise it can't know how many pips you have, or what kind of chests it uses)

It's a bit weird though that it's been a week since it started and it's still not coming back.... maybe I'm doing something wrong, but I manually checked and it's not there: https://api.guildwars2.com/v2/pvp/seasons gives the existing season `id`s, then grabbing the last one I can get the season data https://api.guildwars2.com/v2/pvp/seasons/5DD4CF6F-C68B-47E2-8926-8A7D0AE78462 which is still the past one.

Something that could be improved though is comunicating this to the user... a message/overlay of "No active season found" or something, instead of the UI just not working at all / crashing.

**Comment by thecaligarmo** [comment]
Yeah, that would be a good idea. Or else currently it looks like the app itself isn't working.
I was going to suggest using achievements as a back-up, but it looks like the API isn't giving the Season 8 achievements either:
https://api.guildwars2.com/v2/achievements/categories?id=248

Sounds like Anet just hasn't released their newest set yet. 

Alternative (less awesome) way of handling it is: if the API isn't returning something, you can have the user manually enter the info? "The API isn't returning anything. When does the current season end?" and then you can do your calculations that way?

**Comment by voliva** [comment]
I'll give this a go 🚀

**Comment by voliva** [comment]
Released an initial version on https://github.com/voliva/gw2-pvp-pips/releases/tag/v0.3.0, give it a try please!

I think the way of entering the current pips you have could be improved... it's a pain to count them by hand. I will have to add in a menu so you can select your current reward chest + pips. Reusing this issue.

Edit: I also have more things to fix :'D I've increased the number of pips received by 2 because on last season I was on platinum and I wanted an accurate measurement, but this shouldn't have made it into the release. It would be great to have the feature of setting this up, but it's not part of this issue.

**Comment by thecaligarmo** [comment]
Looking better!
Some minor notes; "All chests" now seems confusing since for 2v2 "All chests" would mean only up to Persimmon, but it still (by default) counts all of them. Maybe "Regular season chests" would be better? (Or just not have a default?)

I agree with you that trying to count # of pips is intense lol Maybe you can do something like what they do for WvW? (https://gw2.limitlessfx.com/wvw/pips.php) That might be an idea if you want. (I think people are ok with it being not perfect anyway, as long as it gives a good enough estimate? Especially when the API isn't working)
